client/flutter/lib/components/home_page_donate.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/cupertino.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:who_app/components/themed_text.dart';
+import 'package:who_app/constants.dart';
+import 'package:who_app/generated/l10n.dart';
+
+class HomePageDonate extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: <Widget>[
+        Positioned.fill(
+          child: _DonateBackground(),
+        ),
+        IntrinsicHeight(
+          child: Padding(
+            padding: EdgeInsets.fromLTRB(0.0, 72.0, 0.0, 60.0),
```

Comment:
if you are using only top/bottom, perhaps use .only

Reply:
Originally, I was handling the padding differently to where I was using the sides, too - now that I am not, I will update 👍